inference-engine/tests/functional/inference_engine/ir_serialization/rt_info_deserialization.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <memory>
+
+#include <inference_engine.hpp>
+#include <transformations/rt_info/fused_names_attribute.hpp>
+
+using namespace ngraph;
+
+TEST(RTInfoDeserialization, Node) {
+    std::string model = R"V0G0N(
+<net name="Network" version="10">
+    <layers>
+        <layer name="in1" type="Parameter" id="0" version="opset8">
+            <data element_type="f32" shape="1,3,22,22"/>
+            <rt_info>
+                <attribute name="fused_names" version="0" value="in1"/>
```

Comment:
can we also check how existing `layout` runtime info will look in the IR? Will it be correctly serialized/deserialized?

Reply:
I would like to have such tests but unfortunately we don't actually have them so they needs to be developed from scratch. Also I don't remember cases when pugi failed to serialize or read xml. If we are talking about rt info section, we have IR reader tests with rt info located inside node, input and output ports and IR serialization tests which are also exploit this functionality.